Composer/packages/client/src/components/ProjectTree/TriggerCreationModal.tsx

```diff
@@ -37,7 +37,7 @@ import { StoreContext } from '../../store';
 
 import { styles, dropdownStyles, dialogWindow, intent } from './styles';
 
-const nameRegex = /^[a-zA-Z0-9-_.]+$/;
+const nameRegex = /^[a-zA-Z0-9-_]+$/;
```

Comment:
This should probably be hoisted into a shared module that can be consumed by both places where it is used.